app/views/redmine_required_field/_settings.html.erb

```diff
@@ -83,15 +91,19 @@
   var rrf_issue_statuses = <%= raw rrf_all_issue_statuses.map(&:reverse).to_h.to_json %>;
   var rrf_trackers = <%= raw rrf_all_trackers.map(&:reverse).to_h.to_json %>;
   var rrf_projects = <%= raw rrf_all_projects.map(&:reverse).to_h.to_json %>;
+  var rrf_roles = <%= raw rrf_all_roles.map(&:reverse).to_h.to_json %>;
 
   function rrf_redraw_issue_required() {
     var table_body = '';
-
     $.each(rrf_current_issue_required, function(ix, data) {
+      console.log(data);
```

Comment:
would be good not to leave console.log around as the users browser may not be happy about it

Reply:
absolutely right.